nexusadspy/segment.py

```diff
@@ -88,11 +88,17 @@ def _get_buffer_for_upload(self):
         return compressed_buffer
 
     def _get_upload_string_for_user(self, user):
+        """
+        The upload string needs to follow the order that has been specified with AppNexus:
+        uid, seg_id, timestamp, expiration, value, member_id, seg_code
```

Comment:
this order is business logic i.e. needs to be fed in through an argument and not hard-coded